services/oc-pmrmrg/versioning/integration.cfg

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 1.0.0
```

Comment:
ok your metadata file is not in sync. here it says 1.0.0 and in the metadata 0.1.0?
maybe there is a cookiecutter issue? but for sure if the metadata is on 0.1.0 it will fail running in simcore

Reply:
I think this happened when I tried to bumpversion and it failed because my repo wasn't clean. However it somehow bumped certain of the versions and not others. I will fix here but it seems not awesome of the bumpversion to handle it like that